src/qml/pages/EditProfilePage.qml

```diff
@@ -0,0 +1,188 @@
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+
+import "../theme"
+import "../components"
+
+//Q_PROPERTY(QString userId READ userId WRITE setUserId NOTIFY userIdChanged)
+//Q_PROPERTY(QString phoneNumber READ phoneNumber WRITE setPhoneNumber NOTIFY phoneNumberChanged)
+//Q_PROPERTY(QString email READ email WRITE setEmail NOTIFY emailChanged)
+//Q_PROPERTY(bool isPhoneNumberConfirmed READ isPhoneNumberConfirmed WRITE setIsPhoneNumberConfirmed NOTIFY isPhoneNumberConfirmedChanged)
+//Q_PROPERTY(bool isEmailConfirmed READ isEmailConfirmed WRITE setIsEmailConfirmed NOTIFY isEmailConfirmedChanged)
+//Q_PROPERTY(QUrl avatarUrl READ avatarUrl WRITE setAvatarUrl NOTIFY avatarUrlChanged)
+
+OperationPage {
+    id: editProfilePage
+    appState: app.stateManager.editProfileState
+    property bool isPhoneConfirmed: appState.isPhoneNumberConfirmed
```

Comment:
alias?

Reply:
Alias don't work this way. It can't find id "appState". I got an Error